analyzers/src/SonarAnalyzer.Common/Rules/Hotspots/RequestsWithExcessiveLengthBase.cs

```diff
@@ -136,7 +136,7 @@ private bool IsEnabled(AnalyzerOptions options)
             return SupportedDiagnostics.Any(d => analyzerConfiguration.IsEnabled(d.Id));
         }
 
-        private struct Attributes
+        private struct Attributes // This struct is used as the same attributes can not be applied multiple times to the same declaration.
```

Comment:
- nitpick: should be on the line above, we avoid inline comments (especially when it's long)